README.md

```diff
@@ -85,6 +85,17 @@ You can run the integration tests with this command:
 $ pytest tests/integration
 ```
 
+### Documentation 
+
+Sphinx is used for documentation. You can generate HTML locally with the following:
+
+```
+$ pip install -r requirements-docs.txt
```

Comment:
We already installed the dev dependencies in L71/72. We can omit it here.

Reply:
Done